### DISCUSSION TITLE
Revise GoDaddy Troubleshooting Wiki

### DISCUSSION THREAD

**Discussion by parparpoundski** [PHPMailer/PHPMailer#2872] [General]
Hello,
    Thanks for all of your contributions with PHPMailer. 

Through a bit of trial and error, I have learned that GoDaddy **really** doesn't like HTML in the message. It will sometimes work and others not. The only way that I was able to get GoDaddy mail's to work was by using the default PHP mail function (not PHPMailer) and not sending **ANY** header information. While I understand that this does not directly effect this project, it may help people reduce the amount of time attempting to get GoDaddy's snail mail to work. I could play around with it more to get it to an exact science if need be.

Thank you.

**Comment by Synchro** [comment]
Thanks! I can only feel sorry for those forced to use GoDaddy! If you have any insights, do notice that the troubleshooting guide is editable by anyone, so you can add you findings in there.

**Reply by bharatptl** [reply]
hi can you give details how to use phpmailer with godaddy ???

**Reply by Synchro** [reply]
Start by reading the troubleshooting guide in the PHPMailer wiki, linked at the top of this page.
